src/lib/support/Span.h

```diff
@@ -62,10 +62,19 @@ class Span
     constexpr explicit Span(U (&databuf)[N]) : mDataBuf(databuf), mDataLen(N)
     {}
 
+    // Creates a (potentially mutable) Span view of an std::array
     template <class U, size_t N, typename = std::enable_if_t<sizeof(U) == sizeof(T) && std::is_convertible<U *, T *>::value>>
     constexpr Span(std::array<U, N> & arr) : mDataBuf(arr.data()), mDataLen(N)
```

Comment:
It would be nice if at least the overload taking a non-const lvalue reference would be `explicit`, but that breaks a lot of places in the code.